blocklearning/lib/core/theme/app_text_styles.dart

```diff
@@ -3,22 +3,22 @@ import 'package:flutter/material.dart';
 
 abstract class AppTextStyles {
```

Comment:
It is what we speak about with Stas when i create different text styles. It's should be made with the help of fabric constructor.

Reply:
i think we don't need in so hard solution